fonts/generate.py

```diff
@@ -83,21 +82,20 @@ def generate_smufl(opts: Namespace) -> bool:
     :param opts: A set of options from the argument parser sub-command.
     :return: True if successful, False otherwise.
     """
-    supported_glyphs_fname: str = opts.supported
-    supported_pth: Path = Path(supported_glyphs_fname)
+    supported_pth: Path = Path(opts.supported)
     header_out_pth: Path = Path(opts.header_out)
     header_file_pth: Path = Path(header_out_pth, "smufl.h")
 
     if not supported_pth.is_file() or not os.access(supported_pth, os.R_OK):
-        log.error("Could not find or read %s file.", supported_glyphs_fname)
+        log.error("Could not find or read %s file.", supported_pth)
```

Comment:
I want the full path as provided so that users can see where it's looking, since we do some implicit path creation based on font names, so having the full path is useful for checking that the file actually exists. 

`.name` only shows the last component of the path, whereas casting a `Path` to a string (or just printing it) will print the full path.

```
from pathlib import Path

p = Path("/foo/bar/baz.txt")
p.name  #=> 'baz.txt'
str(p)     #=> '/foo/bar/baz.txt'
f"{p}"     #=> '/foo/bar/baz.txt'
```